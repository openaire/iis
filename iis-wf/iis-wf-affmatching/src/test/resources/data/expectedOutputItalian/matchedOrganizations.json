{
    "documentId":"50|doi_________::d36eeb1f95335af209276f15e052a51f",
    "organizationId":"20|corda__h2020::253b98253327b261bc052f3fb00d7b44",
    "matchStrength":0.964
}
{
    "documentId":"50|doi_________::d36eeb1f95335af209276f15e052a51f",
    "organizationId":"20|erasmusplus_::cefb3c8215811235a8c851532c96a3cf",
    "matchStrength":0.964
}
{
    "documentId":"50|doi_________::d36eeb1f95335af209276f15e052a51f",
    "organizationId":"20|doajarticles::ade4d79c6a504795d1fcb3c10b218a2c",
    "matchStrength":0.988
}