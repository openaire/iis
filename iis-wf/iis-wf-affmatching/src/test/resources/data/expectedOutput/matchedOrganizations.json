{
    "documentId":"ABC",
    "organizationId":"20|UW",
    "matchStrength":0.9999995
}
{
    "documentId":"20002AX",
    "organizationId":"20|UW",
    "matchStrength":0.9999995
}
{
    "documentId":"DOC_3",
    "organizationId":"20|UCB",
    "matchStrength":1.0
}
{
    "documentId":"DOC_3",
    "organizationId":"20|ICMUW",
    "matchStrength":0.99997675
}
{
    "documentId":"DOC_4",
    "organizationId":"20|UCB",
    "matchStrength":1.0
}
{
    "documentId":"DOC_6",
    "organizationId":"20|WSU",
    "matchStrength":0.999957
}
{
    "documentId":"DOC_6",
    "organizationId":"20|WSU_DUP",
    "matchStrength":0.999957
}